content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -644,7 +644,7 @@ in Go files or in the
 | Golang marker | OpenAPI extension | Accepted values | Description | Introduced in |
 |---|---|---|---|---|
 | `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | Applicable to lists. `atomic` and `set` apply to lists with scalar elements only. `map` applies to lists of nested types only. If configured as `atomic`, the entire list is replaced during merge; a single manager manages the list as a whole at any one time. If `granular`, different managers can manage entries separately. | 1.16          |
-| `//+listMapKeys` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. | 1.16 |
+| `//+listMapKey` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. While there can be multiple keys, `listMapKey` is singular because keys need to be specified individually in the Go type. | 1.16 |
```

Comment:
nit: I am reading the corresponding code for the first time. Is the comment explaining why the name is singular versus plural and this is related to the code?
The `//+listMapKey` marker name is singular instead of plural. You must set each key individually ... ?

Reply:
Just to make sure we're all on the same page, one is supposed to repeat the marker with each key in go.

In go it looks like this:
```golang
// +listMapKey=name
// +listMapKey=protocol
```

The generated OpenAPI extensions looks like:
```
"x-kubernetes-list-map-keys": ["name", "protocol"]
```

Reply:
OK, thanks for the explanation.
/lgtm